azkaban-execserver/src/main/java/azkaban/execapp/ServerStatisticsServlet.java

```diff
@@ -67,82 +67,114 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
    * a double value will be used to present the remaining memory,
    *         a returning value of '55.6' means 55.6%
    */
-  protected void fillRemainingMemoryPercent(ExecutorInfo stats){
-    if (new File("/bin/bash").exists()
-        && new File("/bin/cat").exists()
-        && new File("/bin/grep").exists()
-        &&  new File("/proc/meminfo").exists()) {
-    java.lang.ProcessBuilder processBuilder =
-        new java.lang.ProcessBuilder("/bin/bash", "-c", "/bin/cat /proc/meminfo | grep -E \"MemTotal|MemFree\"");
-    try {
-      ArrayList<String> output = new ArrayList<String>();
-      Process process = processBuilder.start();
-      process.waitFor();
-      InputStream inputStream = process.getInputStream();
+  protected void fillRemainingMemoryPercent(ExecutorInfo stats) {
+    if (new File("/bin/bash").exists() && new File("/bin/cat").exists() && new File("/bin/grep").exists()
```

Comment:
Does this check need to happy every time this method is called?  Can you do this check once in the constructor?